CHANGELOG.md

```diff
@@ -4,6 +4,22 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format located [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
 ## [Unreleased]
+### Added
+- Most resources now support metadata specific properties (@webframp)
+
+### Changed
+- *Breaking* `sensu_environment` is now `sensu_namespace` (@webframp)
```

Comment:
these shold all be under the `### Breaking Changes` header

Reply:
Moved these to a different header